.circleci/config.yml

```diff
@@ -956,59 +982,4 @@ jobs:
 workflows:
   build-test:
     jobs:
-      - flake8
```

Comment:
Why are all these jobs deleted? Was that just for testing this change?

Reply:
Sorry, just for testing.